docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,26 @@
+# Troubleshooting
+
+## Known Problems + Solutions
+
+1) If you uninstall Karpenter after creating some nodes with Karpenter, you will be left with nodes that have finalizers that need to be patched out for the node object to be deleted.
```

Comment:
Consider making this problem it's own `## Unable to delete Nodes after uninstalling Karpenter`